### PR TITLE
add SteamAuthenticationBundle recipe

### PR DESCRIPTION
**Pull request by knojector**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by knojector** [comment]
The Bundle requires the security component but refering to the contribution guideline it's not valid if the bundle requires the component. Is there a solution for this?

The 3.4 Travis fails because of the required minimum version of Symfony 4.0.

File: knojector/steam-authentication-bundle/1.0/config/packages/knojector_steam_authentication.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+knojector_steam_authentication:
+    api_key: ''
```
Comment (Nyholm) [inline knojector/steam-authentication-bundle/1.0/config/packages/knojector_steam_authentication.yaml]: What about adding environment parameter here?

File: knojector/steam-authentication-bundle/1.0/config/packages/knojector_steam_authentication.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+knojector_steam_authentication:
```
Comment (Nyholm) [inline knojector/steam-authentication-bundle/1.0/config/packages/knojector_steam_authentication.yaml]: Maybe a link to your documentation so I know where to go if I need support/docs?

File: knojector/steam-authentication-bundle/1.0/config/packages/knojector_steam_authentication.yaml [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+    api_key: ''
+    login_route: ''
+    login_redirect: ''
+    user_class: ''
```
Comment (Nyholm) [inline knojector/steam-authentication-bundle/1.0/config/packages/knojector_steam_authentication.yaml]: route, redirect and user_class are all require right?

Comment (knojector) [reply knojector/steam-authentication-bundle/1.0/config/packages/knojector_steam_authentication.yaml]: Yes, they are all required. Do I have to make any changes to this?

Comment (Nyholm) [reply knojector/steam-authentication-bundle/1.0/config/packages/knojector_steam_authentication.yaml]: No, you are fine. Thanks

**Comment by Nyholm** [comment]
This looks good. Thanks!

**Pull request merged**